JNWCollectionView/JNWCollectionViewFramework.m

```diff
@@ -131,9 +137,23 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 	return self;
 }
 
+#pragma mark Notifications
+
+- (void)preferredScrollerStyleDidChangeNotification:(NSNotification *)note {
+	NSView *documentView = self.documentView;
+	
+	if (!CGSizeEqualToSize(NSSizeToCGSize(documentView.frame.size), NSSizeToCGSize(self.contentSize))) {
+		CGRect documentFrame = documentView.frame;
+		documentFrame.size.width = self.contentSize.width;
+		documentView.frame = documentFrame;
```

Comment:
I am iOS dev so i even didn't know about layoutDocumentView, in free time i will try replace it wigh layoutDocumentView, btw. cell frame are based on document view frame so anyway shouldn't invalidate layout be after layout document view?